correctedit.py

```diff
@@ -1,21 +1,21 @@
-from gingerit.gingerit import GingerIt
+from gingerit.gingerit import GingerIt #Gingerit is used to ease the file collection of dictionaries and to compare the data to the database.

```

Comment:
Hey Gokul, I can see that we have comments now in the program.

Great! Will merger it now.